docker-app/qfieldcloud/core/filters.py

```diff
@@ -0,0 +1,44 @@
+import django_filters
+from django.db import models
+
+from qfieldcloud.core.models import Project, ProjectQueryset
+
+
+class IncludePublicChoices(models.IntegerChoices):
+    EXCLUDE = 0, "Exclude"
+    INCLUDE = 1, "Include"
+
+
+class ProjectFilterSet(django_filters.FilterSet):
+    owner = django_filters.CharFilter(field_name="owner__username", lookup_expr="exact")
+    include_public = django_filters.ChoiceFilter(
+        label="Include public projects",
```

Comment:
```suggestion
        label="Include public projects. Can be provided with it's deprecated name `include-public`.",
```

Or something better, you name it.

Reply:
Done.